packages/sites/genomics-site/webapp/wdkCustomization/js/client/components/records/ResultExportSelector.tsx

```diff
@@ -131,40 +126,36 @@ function useStyles<T extends string, S, E>(
                 transform: 'scale(0.98)',
                 WebkitTransform: 'scale(0.98)',
                 boxShadow: '0 2px 5px rgb(0 0 0 / 5%)',
-              }
+              },
             }
           : {
               cursor: 'not-allowed',
               borderWidth: '0.5px 0.5px 3px 0.5px',
-            }
-      ),
-    }),
-    indicatorSeparator: (baseStyles) => ({
-      ...baseStyles,
-      display: 'none',
-    }),
-    dropdownIndicator: (baseStyles, dropdownIndicatorProps) => ({
-      ...baseStyles,
-      padding: '0 8px',
-      ...(
-        !dropdownIndicatorProps.isDisabled
+            }),
+      }),
+      indicatorSeparator: (baseStyles) => ({
+        ...baseStyles,
+        display: 'none',
+      }),
+      dropdownIndicator: (baseStyles, dropdownIndicatorProps) => ({
+        ...baseStyles,
+        padding: '0 8px',
+        ...(!dropdownIndicatorProps.isDisabled
           ? {
               color: '#3e3e3e',
             }
-          : {}
-      )
+          : {}),
+      }),
+      menu: (baseStyles) => ({
+        ...baseStyles,
+        width: '20em',
```

Comment:
I spy the change!

We really need to figure out a bulk formatting PR to get rid of these annoying diffs!

Reply:
Yes, we should!